start_miner.sh

```diff
@@ -22,4 +22,5 @@ pm2 start neurons/miner.py --name bitmind_miner -- \
   --wallet.name $WALLET_NAME \
   --wallet.hotkey $WALLET_HOTKEY \
   --axon.port $MINER_AXON_PORT \
-  --blacklist.force_validator_permit $BLACKLIST_FORCE_VALIDATOR_PERMIT
+  --blacklist.force_validator_permit $BLACKLIST_FORCE_VALIDATOR_PERMIT \
+  --logging.debug
```

Comment:
do we want to run in debug mode by default?

Reply:
info is better actually, switching to that now